backend/ee/danswer/danswerbot/slack/blocks.py

```diff
@@ -0,0 +1,22 @@
+from slack_sdk.models.blocks import ActionsBlock
+from slack_sdk.models.blocks import Block
+from slack_sdk.models.blocks import ButtonElement
+from slack_sdk.models.blocks import SectionBlock
+
+from danswer.danswerbot.slack.constants import GENERATE_ANSWER_BUTTON_ACTION_ID
+
+
+def build_standard_answer_blocks(
```

Comment:
honestly I would just throw it in the other module just for simplicity (handle_standard_answers.py under ee)

Reply:
Done